spec/factories/registered_user_factory.rb

```diff
@@ -1,13 +1,10 @@
 FactoryBot.define do
-  factory :registered_user do
-    email "user-#{SecureRandom.uuid}@example.com"
+  factory :registered_user, aliases: [:user] do
+    email { "user-#{SecureRandom.uuid}@example.com" }
```

Comment:
TURNS OUT if you don't pass a block this gets evaluated at runtime, so using the step multiple times doesn't work.